src/server/controllers/UserTransaction.controller.ts

```diff
@@ -0,0 +1,175 @@
+import prisma from '@/lib/database/client';
+import { CreateUserDTO } from '@/lib/models/dto/CreateUser.dto';
```

Comment:
I did a big update in the models, so they were renamed, take some time to import again them